weDevsOfficial/dokan
# Enhance/prevent load of unnecessary scripts styles

### All Submissions:






### Changes proposed in this Pull Request:
Stopped loading of certain scripts sitewide. Now those scripts loads on those pages where they are necessary.

this pr is part of [this](https://github.com/weDevsOfficial/dokan-pro/pull/1850) dokan pro pr
Closes #1334 
### How to test the changes in this Pull Request:

1. take a look at [this](https://docs.google.com/spreadsheets/d/1kh2vAXHQyt41gsJkWKt4xgTqK1qdIf0SimhEZBGISOc/edit#gid=1792838472) excel sheet
2. check if the handles specified load sitewide or not
3. check if the handles are loading on required pages by looking at the documentation of Dokan



### Changelog entry

> **enhance:** stop loading unnecessary scripts and styles sitewide


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.
